config/config.go

```diff
@@ -89,6 +89,10 @@ const MaxGenesisIDLen = 128
 // MaxEvalDeltaTotalLogSize is the maximum size of the sum of all log sizes in a single eval delta.
 const MaxEvalDeltaTotalLogSize = 1024
 
+// ForceCatchpointFileGenerationTrackingMode defines the CatchpointTracking mode that would be used to
+// force a node to generate catchpoint files.
+const ForceCatchpointFileGenerationTrackingMode = 99
```

Comment:
I will do, but I don't think I'd call it a new option. It's existing, just undocumented--on purpose I guess?

Reply:
Hmm, good point. Maybe @algorandskiy has an opinion here. Any reason we shouldn't document 99?

Reply:
yes, it intended for internal use/testing. For example, when we disable catchpoints with a release until the consensus hits. We do not want people ever seeing this option otherwise there are would be different catchpoint labels for the same rounds.
I personally would not even put this 99 constant here but instead handle in tracker to keep it internal.

Reply:
Ok I've removed it from the config and docs and put the logic solely in the catchpointtracker. This has the side effect that `StoresCatchpoints` and `TracksCatchpoints` don't always represent the behaviour of the node now since their expectations get overridden within the tracker itself.